packages/pigment-css-react/src/Stack.d.ts

```diff
@@ -12,6 +12,8 @@ type StackBaseProps = {
   className?: string;
 };
 
-declare const Stack: PolymorphicComponent<StackBaseProps>;
```

Comment:
Can you add a typecheck test for the change if possible ?

Reply:
Done